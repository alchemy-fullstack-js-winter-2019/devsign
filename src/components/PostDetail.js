import React from 'react';
import Header from './Header';
import Footer from './Footer';

export default function PostDetail() {
  return (
    <>
      <Header/>
      <section>
        <h3>Post Detail</h3>
        <div>
          <h4>Post Title</h4>
          <p>aut dicta possimus sint mollitia voluptas commodi quo doloremque iste corrupti reiciendis voluptatem eius rerum sit cumque quod eligendi laborum minima perferendis recusandae assumenda consectetur porro architecto ipsum ipsam.</p>
        </div>
      </section>
      <Footer/>
    </>  
  );
}
