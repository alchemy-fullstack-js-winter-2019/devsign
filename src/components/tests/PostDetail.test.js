import React from 'react';
import renderer from 'react-test-renderer';
import PostDetail from '../PostDetail';
import { MemoryRouter } from 'react-router';

describe('Post Detail snapshot', () => {
  it('can match a snapshot', () => {
    const tree = renderer.create(
      <MemoryRouter>
        <PostDetail />
      </MemoryRouter>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
