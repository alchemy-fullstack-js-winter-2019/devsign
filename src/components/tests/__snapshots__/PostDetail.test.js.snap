// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Post Detail snapshot can match a snapshot 1`] = `
Array [
  <p>
    My Blog
  </p>,
  <nav>
    <span>
      <a
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </span>
    <span>
      <a
        href="/login"
        onClick={[Function]}
      >
        Login
      </a>
    </span>
    <span>
      <a
        href="/posts"
        onClick={[Function]}
      >
        Posts
      </a>
    </span>
    <span>
      <a
        href="/about"
        onClick={[Function]}
      >
        About
      </a>
    </span>
  </nav>,
  <section>
    <h3>
      Post Detail
    </h3>
    <div>
      <h4>
        Post Title
      </h4>
      <p>
        aut dicta possimus sint mollitia voluptas commodi quo doloremque iste corrupti reiciendis voluptatem eius rerum sit cumque quod eligendi laborum minima perferendis recusandae assumenda consectetur porro architecto ipsum ipsam.
      </p>
    </div>
  </section>,
  <p>
    Copyright 2019
  </p>,
]
`;
